### PR TITLE
Added compatibility note to clouds-maas

### PR DESCRIPTION
**Pull request by evilnick**
Part of work for #1981

### CODE REVIEW

File: src/en/clouds-maas.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ Title: Using MAAS with Juju
 # Using a MAAS cloud
 
 Juju works closely with [MAAS][maas-site] to deliver the same experience
-on bare metal that you would get by using any other cloud.
+on bare metal that you would get by using any other cloud. Note that the 
+Juju 2.x software is compatible with both the 1.x and 2.x series of MAAS.
```
Comment (pmatulis) [inline src/en/clouds-maas.md]: Awkward.

Maybe:

"Note that the Juju 2.x series is compatible..."

Comment (evilnick) [reply src/en/clouds-maas.md]: fine

**Comment by evilnick** [comment]
@pmatulis if the PR is from someone on the docs team, and you trust them to make requested changes, I think it is okay to follow what I now call the " @degville protocol" - make the comments inline but if there is no need to check the changes, give it an "approved" anyhow. I think everyone is sensible enough not to merge things which require a further look. This for example, would save you the time of coming back here and clicking on things to see that I changed one word.

**Comment by pmatulis** [comment]
I forgot to click "Approve". That's all.

**Pull request merged**